<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget_udp" native="true">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>461</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>消息记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="recordEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>351</width>
        <height>301</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>223</width>
         <height>270</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="localPort"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>本地端口</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="aimPort"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>目标端口</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="aimIP"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>目标IP</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="localIP"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>本地IP</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="sendEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>发送框</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="count">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>接收/发送</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>249</x>
         <y>9</y>
         <width>91</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="openBtn">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetnumBtn">
          <property name="text">
           <string>重新计数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_control" native="true">
     <widget class="QWidget" name="widget_data_record" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>255</width>
        <height>522</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>48</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数据记录</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="data_record_edit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>27</y>
         <width>237</width>
         <height>486</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_setting" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>241</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>波段</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="band_comboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Ka</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-18GHz</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>通道</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="channel_comboBox">
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>通道频率</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="channel_freq">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>支路</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="branch_comboBox">
         <item>
          <property name="text">
           <string>A支路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B支路</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C支路</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>一级开关</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="first_switch"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>二级开关</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="second_switch"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>三级开关</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="third_switch"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>衰减器1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="first_attenuator"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>衰减器2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="second_attenuator"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>移相器</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QLineEdit" name="phaser"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>延时器</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QLineEdit" name="delay"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>校准</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="QLineEdit" name="calibrate"/>
       </item>
       <item row="12" column="2">
        <widget class="QPushButton" name="convert_btn">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_btn" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>537</y>
        <width>255</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="array_setting_btn">
         <property name="text">
          <string>阵列设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calibration_setting_btn">
         <property name="text">
          <string>校准设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_data_btn">
         <property name="text">
          <string>读取数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_data_btn">
         <property name="text">
          <string>清除数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QRadioButton" name="rough_ctrl">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>450</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>粗控</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="accurate_ctrl">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>480</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>精控</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
